### ISSUE TITLE
Deepannotate model

### ISSUE THREAD

**Issue by mohamedsamirx** [naurril/SUSTechPOINTS#215]
hello, i wanted to deploy the yaw prediction (deep annotate model) using tfjs but got unsupported layer 
tfUnknown layer: TensorFlowOpLayer. This may be due to one of the following reasons:
1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.
2. The custom layer is defined in JavaScript, but is not registered properly with tf.serialization.registerClass().
    at deserializeKerasObject (generic_utils.js:227:1)

is there a way to solve that, or the only way is to register the model with custom layers ?

**Comment by mohamedsamirx** [comment]
i know that the problem is because the model is  built using tensorflow 2.10.0 but is there solution that i may now know about to solve that, like custom plugins or etc ? also if i toke the deepannotate repo code update it and train the model again will that solve the issue or it will continue ??

**Comment by naurril** [comment]
the `tfUnknown` problem is probably caused by the loss-header of the training model. If my guess is right then re-train and export the model without the header could probably solve this problem

**Comment by mohamedsamirx** [comment]
Okay thank you
